independent-publisher-connectors/Mensagia/apiProperties.json

```diff
@@ -1,25 +1,25 @@
 {
-  "properties": {
-    "connectionParameters": {
-      "api_key": {
-        "type": "securestring",
-        "uiDefinition": {
-          "displayName": "Api Key",
-          "description": "The API Key for this api",
-          "tooltip": "Provide your API Key",
-          "constraints": {
-            "tabIndex": 2,
-            "clearText": false,
-            "required": "true"
-          }
-        }
-      }
-    },
-    "iconBrandColor": "#da3b01",
-    "capabilities": [
-      "actions"
-    ],
-    "publisher": "Sistemas Informaticos ICON, S.L.",
-    "stackOwner": "Sinergyne Global Communications, S.L."
-  }
-}
+    "properties": {
+        "connectionParameters": {
+            "api_key": {
+                "type": "securestring",
+                "uiDefinition": {
+                    "displayName": "Api Key",
+                    "description": "Api Key for this API.",
+                    "tooltip": "Set your Api Key.",
+                    "constraints": {
+                        "tabIndex": 2,
+                        "clearText": false,
+                        "required": "true"
+                    }
+                }
+            }
+        },
+        "iconBrandColor": "#FDFDFD",
```

Comment:
@Sistemes-Icon Please follow the template provided for apiProperties.json for IP connectors, it should be default for all the IP connectors and that is `#da3b01`